examples/TestAppUwp/NodeDssSignaler.cs

```diff
@@ -77,6 +77,26 @@ public static WireMessageType WireMessageTypeFromString(string stringType)
                 throw new ArgumentException($"Unkown signaler message type '{stringType}'");
             }
 
+            public static WireMessageType TypeFromSdpMessageType(SdpMessageType type)
+            {
+                switch (type)
+                {
+                case SdpMessageType.Offer: return WireMessageType.Offer;
+                case SdpMessageType.Answer: return WireMessageType.Answer;
+                }
```

Comment:
nit:
```suggestion
                switch (type)
                {
                    case SdpMessageType.Offer: return WireMessageType.Offer;
                    case SdpMessageType.Answer: return WireMessageType.Answer;
                }
```

Reply:
Ignoring for now, there's another change coming that clean-up that.